pkg/sqlmigrations/migrations.go

```diff
@@ -1262,7 +1273,7 @@ func createNewSystemNamespaceDescriptor(ctx context.Context, r runner) error {
 		//    the correct ID in the new system.namespace table after all tables are
 		//    copied over.
 		nameKey := sqlbase.NewPublicTableKey(
-			sqlbase.NamespaceTable.GetParentID(), sqlbase.NamespaceTable.GetName())
+			sqlbase.NamespaceTable.GetParentID(), sqlbase.NamespaceTableName)
```

Comment:
This change seems wrong -- we want to write the new tables name, not 'namespace' here